docs/library/xilinx/clk_vip/index.rst

```diff
@@ -0,0 +1,119 @@
+:orphan:
+
+.. _template_ip:
```

Comment:
Maybe xilinx_template_ip? Because this will definitely overlap with the other template_ip from the same repo